apps/mpi-job/upstream/base/deployment.yaml

```diff
@@ -1,27 +0,0 @@
-apiVersion: apps/v1
```

Comment:
I think MPIJob will not be part of Unify Training operator in Kubeflow 1.4, so we should keep these manifests.

cc @kubeflow/wg-training-leads

Reply:
sg

Reply:
@mkbhanda @DavidSpek @kimwnasptd @RFMVasconcelos 

It was brought up during the community meeting Aug 17th that MPI-Operator release should be kept separate for the 1.4 release

Reply:
# MPI Operator
- ../apps/mpi-job/upstream/overlays/kubeflow

Let's keep this one